Assets/LeapMotionModules/ElementRenderer/Resources/LeapGui.cginc

```diff
@@ -0,0 +1,3 @@
+#include "UnityCG.cginc"
+
+#define ELEMENT_MAX 32
```

Comment:
And yeah, currently a max of 32, but that is an arbitrary number that is GPU side only.  It can be increased here without any need to change anything CPU side (for now)